### PR TITLE
Pause on comment feature

### PR DESCRIPTION
**Pull request by kaixiw**

### CODE REVIEW

File: content/modules/dev/ui.editorview.js [hunk 1]
```diff
@@ -66,14 +68,23 @@
...
                     self._selection        = evt.value.selection;
-                    self._html5range    = evt.value.html5range; 
+                    self._html5range    = evt.value.html5range;
                     self._sel        = null;
                     self._note        = null;
                     model.add("draft", drafts);
                     self._render(id_item, evt.value.suppressFocus);
-                    break;
-                case "reply_thread": 
+<<<<<<< HEAD
```
Comment (kpeng94) [inline content/modules/dev/ui.editorview.js]: Is this supposed to be here?

Comment (karger) [inline content/modules/dev/ui.editorview.js]: looks like a merge conflict that wasn't properly resolved?

Comment (kaixiw) [inline content/modules/dev/ui.editorview.js]: My bad! I pushed the fixed version. Thanks for checking!

**Comment by karger** [comment]
This pull request doesn't include any (new) migrations.  How are the changes to the database supposed to happen?

**Comment by karger** [comment]
I merged this branch into my dev branch and ran makemigrations and migrate to update the database.  That seemed to work fine.  The pause on Video comments feature is visible in the properties page of the class.  However, when I (staff) create a comment, no pause happens.

**Comment by karger** [comment]
Suppose I create a staff comment *then* enable pause on staff comments by default.  Will the comment pause?  Conversely, suppose I have pause by default enabled and create a pausing comment.  If I disable pause by default will I still pause at that comment?

**Comment by kaixiw** [comment]
This is what should be happening (please let me know if it isn't working for you): 
A checkbox underneath the duration textbox appears when a staff makes a comment. If the checkbox is checked, then the video will pause when it reaches the comment. When pause on default is enabled, the checkbox should be automatically checked when you make a comment. 

If you create a staff comment, then set the 'pause on default' setting to true, the previous comment will not pause the video. It all depends on whether the pause setting is checked for the individual comment.

**Pull request merged**